.storybook/test-results.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"numFailedTestSuites":0,"numFailedTests":0,"numPassedTestSuites":2,"numPassedTests":8,"numPendingTestSuites":0,"numPendingTests":0,"numRuntimeErrorTestSuites":0,"numTodoTests":0,"numTotalTestSuites":2,"numTotalTests":8,"openHandles":[],"snapshot":{"added":0,"didUpdate":false,"failure":false,"filesAdded":0,"filesRemoved":0,"filesUnmatched":0,"filesUpdated":0,"matched":0,"total":0,"unchecked":0,"uncheckedKeysByFile":[],"unmatched":0,"updated":0},"startTime":1558619536452,"success":true,"testResults":[{"assertionResults":[{"ancestorTitles":[],"failureMessages":[],"fullName":"renders without crashing","location":null,"status":"passed","title":"renders without crashing"}],"endTime":1558619538250,"message":"","name":"/Users/enzo/repositories/viaa/avo2-components/src/components/Button/Button.test.tsx","startTime":1558619537307,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["<Icon />"],"failureMessages":[],"fullName":"<Icon /> Should render when a known icon is passed","location":null,"status":"passed","title":"Should render when a known icon is passed"},{"ancestorTitles":["<Icon />"],"failureMessages":[],"fullName":"<Icon /> Should not render when an unknown icon is passed","location":null,"status":"passed","title":"Should not render when an unknown icon is passed"},{"ancestorTitles":["<Icon />"],"failureMessages":[],"fullName":"<Icon /> Should render icon when name was passed as PascalCase","location":null,"status":"passed","title":"Should render icon when name was passed as PascalCase"},{"ancestorTitles":["<Icon />"],"failureMessages":[],"fullName":"<Icon /> Should render icon when name was passed as kebab-case","location":null,"status":"passed","title":"Should render icon when name was passed as kebab-case"},{"ancestorTitles":["<Icon />"],"failureMessages":[],"fullName":"<Icon /> Should correctly set the className without type","location":null,"status":"passed","title":"Should correctly set the className without type"},{"ancestorTitles":["<Icon />"],"failureMessages":[],"fullName":"<Icon /> Should correctly set the className with type","location":null,"status":"passed","title":"Should correctly set the className with type"},{"ancestorTitles":["<Icon />"],"failureMessages":[],"fullName":"<Icon /> Should correctly set the className with type","location":null,"status":"passed","title":"Should correctly set the className with type"}],"endTime":1558619538593,"message":"","name":"/Users/enzo/repositories/viaa/avo2-components/src/components/Icon/Icon.test.tsx","startTime":1558619537305,"status":"passed","summary":""}],"wasInterrupted":false}
```

Comment:
moet dit gecommit worden of wordt dit automatisch gegenereerd?

Reply:
Auto-generated & moet gecommit worden